library/core/src/time.rs

```diff
@@ -48,6 +48,16 @@ const MICROS_PER_SEC: u64 = 1_000_000;
 ///
 /// let ten_millis = Duration::from_millis(10);
 /// ```
+///
+/// # Formatting `Duration` values
+///
+/// `Duration` intentionally does not have a `Display` impl, as there are a
+/// variety of ways to format spans of time for human readability. `Duration`
+/// provides a `Debug` impl that shows the full precision of the value. The
+/// `Debug` output uses the Unicode "µs" suffix for microseconds; if your
+/// program output may appear in contexts that cannot rely on full Unicode
+/// compatibility, you may wish to format `Duration` objects yourself or use a
+/// crate to do so.
```

Comment:
Should we suggest a crate instead of letting the user to find out themselves?

Reply:
I'm under the impression that we tend to avoid pointing to specific crates in the ecosystem, both because the right crate to use may change, and because the authors of those crates may not want that much of a spotlight on their specific crate.